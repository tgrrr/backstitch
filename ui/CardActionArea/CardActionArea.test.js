import React from 'react'
import renderer from 'react-test-renderer'
import { CardActionArea } from '.'

it('CardActionArea: default', () => {
    const component = renderer.create(
        <CardActionArea>
            <>
                CardActionArea
            </>
        </CardActionArea>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
