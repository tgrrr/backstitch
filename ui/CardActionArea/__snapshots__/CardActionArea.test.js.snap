// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardActionArea: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Cardactionarea "
    data-testid="Cardactionarea"
  >
    <button
      className="MuiButtonBase-root MuiCardActionArea-root"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      CardActionArea
      <span
        className="MuiCardActionArea-focusHighlight"
      />
    </button>
  </div>
</div>
`;
